frontend/packages/dev-console/src/components/add/AddCardItem.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { useUserSettings } from '@console/shared';
+import { SimpleListItem, Title, Text } from '@patternfly/react-core';
+import { ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
+import { navigateTo, resolvedHref } from '../../utils/add-page-utils';
+import './AddPage.scss';
+
+type AddCardItemProps = {
+  action: ResolvedExtension<AddAction>;
+  namespace: string;
+};
+
+const AddCardItem: React.FC<AddCardItemProps> = ({
+  action: {
+    properties: { id, label, icon, href, description },
+  },
+  namespace,
+}) => {
+  const [showDetails] = useUserSettings('console.addPage.showDetails', 'show', true);
+  const actionIcon = (): JSX.Element => {
+    if (typeof icon === 'string') {
+      return <img className="odc-add-page__card-item-header-icon" src={icon} alt={label} />;
+    }
+    if (typeof icon !== 'string' && React.isValidElement(icon)) {
+      return <span className={'odc-add-page__card-item-header-icon'}>{icon}</span>;
```

Comment:
Same here. I would not expect that an icon add significant infos on a screen reader:

```suggestion
      return <span className={'odc-add-page__card-item-header-icon'} aria-hidden="true">{icon}</span>;
```